src/components/Navbar/index.tsx

```
@@ -334,23 +344,22 @@ const Navbar = () => {
-                    alt={item.label}
-                    height="24"
-                    width="24"
-                  />
-                  <NavButtonLabel>{item.label}</NavButtonLabel>
-                </NavButton>
-              </StyledLink>
-            ))}
+            ].map((item) => {
+              const Item = item.icon;
+              return (
+                <StyledLink to={item.href}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)